README.md

```diff
@@ -1,5 +1,163 @@
 # cosy-test
 [![Build Status](https://travis-ci.org/feedzai/cosy-test.svg?branch=master)](https://travis-ci.org/feedzai/cosy-test)
 [![codecov](https://codecov.io/gh/feedzai/cosy-test/branch/master/graph/badge.svg)](https://codecov.io/gh/feedzai/cosy-test)
+[![Maven metadata URI](https://img.shields.io/maven-metadata/v/http/central.maven.org/maven2/com/feedzai/cosy-test_2.12/maven-metadata.xml.svg)](https://mvnrepository.com/artifact/com.feedzai/cosy-test)
 
-Bringing up Docker Compose environments for system, integration and performance testing, with support for ScalaTest and Gatling.
+Bringing up Docker Compose environments for system, integration and performance testing, with support for [ScalaTest](http://www.scalatest.org/),
+[JUnit](https://junit.org/junit4/) and [Gatling](https://gatling.io/).
```

Comment:
Make it explicit that it's JUnit 4 not JUnit5. 

_Nice-to-have improvement_: Modify the DockerComposeRule to make the before() and after() methods simply call some bootstrap and teardown method that is accessible to the library user such that supporting JUnit5 and even TestNG becomes trivial.